tools/pvm/main.cpp

```diff
@@ -0,0 +1,6 @@
+#include <iostream>
+
+int main() {
+  std::cout << "PVM" << std::endl;
```

Comment:
Please, enable this functional only if `LGBTQ` defined. Our VM should not talk with "patriarchalnye ugnitately"!

Reply:
K